runtime/compiler/control/J9Options.cpp

```diff
@@ -981,13 +981,13 @@ TR::OptionTable OMR::Options::_feOptions[] = {
    {"verbose=",           "L{regex}\tlist of verbose output to write to vlog or stdout",
         TR::Options::setVerboseBitsInJitPrivateConfig, offsetof(J9JITConfig, privateConfig), 0, "F"},
    {"version",            "L\tdisplay the jit build version",
-        TR::Options::versionOption, 0, 0},
+        TR::Options::versionOption, 0, 0, "F"},
```

Comment:
Isn't the second `0` supposed to be replaced with the `"F"?

Reply:
I don't think so. If you take a look at the `"verbose="` option above and the actual definition of `TR::OptionTable` [1] you will see the order is, `name`, `helpText`, `fcn`, `parm1`, `parm2`, and then `msg` which is not specified here (so assumed `NULL` - see [2] for a live example).

[1] https://github.com/eclipse/omr/blob/7cf542c14aa04cc5e5baafa4cd88a7b3fbef6f98/compiler/control/OptionsUtil.hpp#L48-L137
[2] https://ideone.com/03Ediv